src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,198 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+    }
```

Comment:
The constructor `MyLinkedList()` is empty and can be omitted unless you plan to include initialization logic in it.